### PR TITLE
Adds icon for sanity

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus (sprites by Mickyan)
fix: sanity can now reach its highest level
imageadd: Adds HUD icons for sanity
/:cl:

### CODE REVIEW

File: code/datums/components/mood.dm [hunk 1]
```diff
@@ -170,15 +173,28 @@
...
+			sanity_level = 4
+		if(SANITY_DISTURBED to SANITY_NEUTRAL)
+			setInsanityEffect(0)
+			master.remove_movespeed_modifier(MOVESPEED_ID_SANITY, TRUE)
+			sanity_level = 3
+		if(SANITY_NEUTRAL+1 to SANITY_GREAT+1) //shitty hack but +1 to prevent it from responding to super small differences
 			setInsanityEffect(0)
 			master.remove_movespeed_modifier(MOVESPEED_ID_SANITY, TRUE)
+			sanity_level = 2
+		if(SANITY_GREAT+1 to INFINITY+1)
```
Comment (Cyberboss) [inline code/datums/components/mood.dm]:
```suggestion
		if(SANITY_GREAT+1 to INFINITY)
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Qustinnus** [comment]
image
what it looks like

**Comment by Militaires** [comment]
isnt the cyan face happier than the green face......... u should make it golden bro

**Comment by DapperMola** [comment]
> isnt the cyan face happier than the green face......... u should make it golden bro

you are telling the creator of /tg/ moodlets that he got the colors wrong.

**Comment by nicbn** [comment]
HUD is already complex enough and no more than the mood indicator is really needed. Not everything should be visible in the HUD.

**Comment by Militaires** [comment]
@DapperMola  are u implying that just because someone made a feature they cant make a mistake related to it 
image

**Comment by Militaires** [comment]
oh hold on this doesnt add a new face with glow for max sanity it adds the ring itself as a meter, nice

**Comment by Mickyan** [comment]
yes, you get an outline around the mood icon for good/bad sanity levels

**Comment by subject217** [comment]
>u should make it golden bro

source material: 
Yuh Yeet

**Pull request merged**